jdaviz/core/astrowidgets_api.py

```diff
@@ -370,6 +370,10 @@ def marker(self, val):
             size = val['markersize']
             if not isinstance(size, (int, float)):
                 raise ValueError(f'Invalid marker size: {size}')
+        if 'fill' in val:
+            fill = val['fill']
+            if not isinstance(fill, bool):
+                raise ValueError(f'Invalid fill: {fill}')

```

Comment:
```suggestion
                raise TypeError(f'Invalid fill: {fill}')

```

Reply:
I am not sure about this. The rest of the `val` checks in that method gives `ValueError`.

Reply:
ok, then I guess consistency is probably the better way to go.